### PR TITLE
As.create navbar

### PR DESCRIPTION
**Pull request by anirudh723**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/677600458/pulses/785273678)

This is for the navigation bar of the LLB frontend. This will allow users to navigate to different parts of the website, for example a page to see their events and a page to update their profile.

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers. 
This required using an appropriate AntDesign component. For this, I decided to go with the Grid Component. Within a row, there are several columns, holding buttons that link to different pages. Also, the authentication state will be received (currently just using a boolean variable that can be turned on/off) to dynamically render the name of the user, or login/signup buttons if the user has not logged in or signed up yet.

## Screenshots

image
image
image

## Verification Steps

Manual verification has been done so far, making sure that the correct information is being displayed upon a button click. Also, interacting with the navigation bar locally helped me confirm that the dropdown is being properly rendered upon a button hover.

### CODE REVIEW

File: src/components/navbar/NavBar.tsx
Comment (rymaju) [inline src/components/navbar/NavBar.tsx:12]: This is the Travis error, we never use setAuthenticated!

File: src/components/navbar/NavBar.tsx
Comment (jackblanc) [inline src/components/navbar/NavBar.tsx:1]: If you rename this file to `index.tsx`, you can import it in App.tsx via: 
`import NavBar from './components/navbar`
Basically if you import a directory using that syntax, it pulls the index.

File: src/components/navbar/navbar.less
Comment (jackblanc) [inline src/components/navbar/navbar.less:6]: I think we're going to want to avoid overwriting ant styles directly based on what I've heard about AntD. 
I've seen a `styled.Button({color: black, margin-left: 7em` syntax before but can't find any docs to link to :( 
@rymaju what's the latest on this? @bensenescu hi :) can you link to docs about that syntax?

File: src/components/navbar/NavBar.tsx
Comment (anirudh723) [reply src/components/navbar/NavBar.tsx:1]: Oooh neat trick. Thanks!

**Comment by rymaju** [comment]
@anirudh723 fixed merge conflicts, this can be merged into master now

File: src/components/navbar/navbar.less
Comment (rymaju) [reply src/components/navbar/navbar.less:6]: Still going to wait on this CSS fix before merge

Comment (anirudh723) [reply src/components/navbar/navbar.less:6]: Yeah I'm a bit unsure on how to go about styling to our standards. I _think_ I may have found what @jackblanc was talking about, and I can try to make that adjustment and push a commit today. But it would be great to get an online resource just to make sure I'm doing what we're trying to do

File: src/components/navbar/index.tsx
Comment (jackblanc) [inline src/components/navbar/index.tsx:37]:
```suggestion
            src="https://lucys-love-bus-public.s3.us-east-2.amazonaws.com/LLB_2019_Sq_rgb+1.png"
            alt="LLB Logo"
```

**Review by jackblanc — APPROVED** [review]

**Pull request merged**
